script/Arcueid/Day 11/11_00_ARC11_3.txt

```diff
@@ -302,7 +302,7 @@ It feels as though this moment is stretching on forever.
 [sha:45b883f4c243c7815a53426991ac50f0b0145bf0]{
 -- Page 174, Offset 30005.
 -- 実際ではおそらく百に満たず秒針が動いて。
-In realty, probably fewer than a hundred seconds expire under the second hand of my watch.
+In the real time, probably fewer than a hundred seconds expire under the second hand of my watch.
```

Comment:
revert

Reply:
Revert but also fix it to be `reality`, I don't think Shiki is renting out the Tohno mansion

Reply:
half reversion